src/govuk/settings/_typography-font.scss

```
@@ -52,6 +52,8 @@ $govuk-font-family-tabular: if(
 
 /// Font families to use for print media
 ///
+/// This option avoids font-related issues that occur with some printer drivers and operating systems.
```

I think it might help to be clear what we mean by 'this' and to explain a bit more about what issues we're avoiding – what about something like:

```suggestion
/// We recommend falling back to system fonts when printing to avoid issues with
/// garbled output seen with some printer drivers / operating systems.
```

---

Hello, not sure how to suggest a change like Ollie has done above. (Enlightenment welcome!)

I agree that it would be good to mention the issues.

Another suggestion: "We recommend that you use system fonts when you print. This will avoid issues with some printer drivers and operating systems."

---

@EoinShaughnessy check out step 6 of https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request 🙂